charts/admission-controller/values.yaml

```diff
@@ -220,26 +220,8 @@ webhook:
     reuseTLSSecret: false
     ca:
       # For outbound connections (secure backend, proxy,...)
-      # <br/>And inbound connections to serve HttpRequests as Kubernetes Webhook.
       # <br/>A PEM-encoded x509 certificate authority.
-      # <br/>If empty, a new CA will be autogenerated.
       cert: ""
-      # For outbound connections (secure backend, proxy,...)
-      # <br/>A PEM-encoded private key of the certificate authority to use in the certificate generation.
-      # <br/>If empty, a new CA will be autogenerated.
-      key: ""
-    # For inbound connections to serve HttpRequests as Kubernetes Webhook.
-    # <br/>A PEM-encoded x509 certificate signed by the CA.
-    # <br/>If empty, a new cert will be generated.
-    # <br/>If provided, it must be valid with the `webhook.ssl.ca`.
-    # <br/>If this is set, the key must also be provided.
-    cert: ""
-    # For inbound connections to serve HttpRequests as Kubernetes Webhook.
-    # <br/>A PEM-encoded private key signed by the CA.
-    # <br/>If empty, a new key will be generated.
-    # <br/>If provided, it must be valid with the `webhook.ssl.ca`.
-    # <br/>If this is set, the cert must also be provided.
-    key: ""
```

Comment:
We can probably keep these `key` inside values since they r going to be used in https://github.com/sysdiglabs/charts/pull/1246/files#diff-154ad88a2fe7629ec86703c63fa030d26afc204bd238ff8a662709298bfc69beL178